service/history/ndc/workflow_resetter.go

```diff
@@ -59,6 +59,14 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+const (
+	maxChildrenInResetMutableState = 1000 // max number of children tracked in reset mutable state
+)
+
+var (
+	errWorkflowResetterMaxChildren = serviceerror.NewInternal(fmt.Sprintf("WorkflowResetter encountered max allowed children [%d] while resetting.", maxChildrenInResetMutableState))
```

Comment:
Invalid argument?

Reply:
What do you mean by invalid arg? I want to include the `maxChildrenInResetMutableState` value in the error message (in case we change these limits).
Or did you mean something else?

Reply:
I mean the error type should be invalid argument, otherwise it counts against SLA.

Reply:
Got it. Changed it to `InvalidArgument`